tasks.ts

```diff
@@ -22,32 +23,36 @@ TaskManager.defineTask(LOCATION_BACKGROUND_TRACKING, async ({ data, error }) =>
   const anyData: any = data;
   if (anyData.locations) {
     const currentLocation: LocationObject = anyData.locations[0];
+    const trackingStart = Number((await readTrackingStart()) ?? '0');
+
+    // Locations with timestamp before tracking started should be discarded
```

Comment:
For some reasons the location updates on physical devices sometimes have timestamps which are multiple hours old. This was the reason for the bug in the score calculation.